### PR TITLE
Show harmony colors pallette in storybook

### LINKED ISSUE #633: Show harmony colors pallette in storybook

### PR DESCRIPTION
**Pull request by DenisVorop**
## PR includes

- [x] Docs

## Related issues

Resolve #633 

## QA Instructions, Screenshots, Recordings
  
Снимок экрана 2023-11-24 в 16 52 35

Снимок экрана 2023-11-24 в 16 52 49

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,7 +79,7 @@
   },
   "dependencies": {
     "@monaco-editor/react": "4.6.0",
-    "@taskany/colors": "1.1.0",
+    "@taskany/colors": "file:../colors",
```
Comment (LamaEats) [inline package.json:82]: um, this isn't works im ci :)

**Review by LamaEats — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (DenisVorop) [reply package.json:82]: I'll wait until a new version of colors is released. Left it so as not to forget

File: .storybook/external/Colors-dark.mdx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import { ColorPalette, ColorItem } from '@storybook/blocks'
+import { getGroupedRootCSSVariables } from '../../src/utils/getGroupedCSSVariables'
+
+<Meta title="Design System/Colors DARK" />
+
+# Color palette for `DARK` theme
+
+<ColorPalette>
+    {
+        Object.entries(getGroupedRootCSSVariables(':root')).map(([title, pallette]) => {
```
Comment (tsumo) [inline .storybook/external/Colors-dark.mdx:11]: I think this also needs an override with `[data-theme="dark"]` for consistency, and to make sure that we are not relying on the default system preference

**Review by tsumo — CHANGES_REQUESTED** [review]

File: .storybook/external/Colors-dark.mdx (hunk 2 above)
Comment (tsumo) [reply .storybook/external/Colors-dark.mdx:11]: But there is no `[data-theme="dark"]` [here](https://github.com/taskany-inc/colors/pull/111) for some reason...

Comment (DenisVorop) [reply .storybook/external/Colors-dark.mdx:11]: `[data-theme="dark"]` used by default

https://github.com/taskany-inc/colors/pull/111/files#diff-4527d57cadc77397bcd394c19dd7e4df5b09df2525420868c661a5f7afafd6b5R1

**Review by tsumo — APPROVED** [review]

**Review by LamaEats — APPROVED** [review]
Please, add darker backgroud color for Colors-dark example

**Pull request merged**